sympy/physics/vector/tests/test_vector.py

```diff
@@ -169,3 +169,11 @@ def test_vector_simplify():
     test4 = ((-4 * x * y**2 - 2 * y**3 - 2 * x**2 * y) / (x + y)**2) * N.x
     test4 = test4.simplify()
     assert (test4 & N.x) == -2 * y
+
+
+def test_vector_evalf():
+    a, b = symbols('a b')
+    v = 5 * A.x
+    assert v.evalf() == 5.0 * A.x
+    v = pi * A.x + 5 * a * A.y - b * A.z
+    assert str(v.evalf(3)) == '3.14*A.x + 5.0*a*A.y - b*A.z'
```

Comment:
I don't think this is the right way to test evalf(). Please check the tests for evalf() and see how it is done there. String comparison doesn't seem appropriate.

Reply:
Majority evalf() tests are string comparison because numeric assertion gives assertion error  even when answer is right.

Reply:
I have changes str to sstr. That's how all evalf test are written.